File: Module1/casestudy/yeucau6/task_1/customer.js
let Customer = function () {
    this.setName = function (name) {
        this.name = name;
    };
    this.getName = function () {
        return this.name;
    };
    this.setCMND = function (cmnd) {
        this.cmnd = cmnd;
    };
    this.getCMND = function () {
        return this.cmnd;
    };
    this.setBirthday = function (birthday) {
        this.birthday = birthday;
    };
    this.getBirthday = function () {
        return this.birthday;
    };
    this.setEmail = function (email) {
        this.email = email;
    };
    this.getEmail = function () {
        return this.email;
    };
    this.setAdress = function (adress) {
        this.adress = adress;
    };
    this.getAdress = function () {
        return this.adress;
    };
    this.setTypeService = function (typeService) {
        this.typeService = typeService;
    };
    this.getTypeService = function () {
        return this.typeService;
    };
    this.setTypeRom = function (typeRom) {
        this.typeRom = typeRom;
    };
    this.getTypeRom = function () {
        return this.typeRom;
    };
    this.setTypeCustomer = function (typeCustomer) {
        this.typeCustomer = typeCustomer;
    };
    this.getTypeCustomer = function () {
        return this.typeCustomer;
    };
    this.setDiscount = function (discount) {
        this.discount = discount;
    };
    this.getDiscount = function () {
        return this.discount;
    };
    this.setQuantilyIncluded = function (quantilyIncluded) {
        this.quantilyIncluded = quantilyIncluded;
    };
    this.getQuantilyIncluded = function () {
        return this.quantilyIncluded;
    };
    this.setRenDays = function (renDays) {
        this.renDays = renDays;
    };
    this.getRenDays = function () {
        return this.renDays;
    };
    this.totalPay = function () {
        let money = 0;
        if (this.getTypeService() === "Villa"){
            money = 500;
        }else if (this.getTypeService() === "House"){
            money = 300;
        }else if (this.getTypeService() === "Room"){
            money = 200;
        }
        return this.getRenDays()*money*(1 - this.getDiscount()/100);
    }



}

File: Module3/bai10_JSTL/bai_tap/display_list_custumer/src/model/Customer.java
package model;

public class Customer {
    private String name;
    private String birthday;
    private String address;
    private String imageLink;

    public Customer() {
    }

    public Customer(String name, String birthday, String address, String imageLink) {
        this.name = name;
        this.birthday = birthday;
        this.address = address;
        this.imageLink = imageLink;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBirthday() {
        return birthday;
    }

    public void setBirthday(String birthday) {
        this.birthday = birthday;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getImageLink() {
        return imageLink;
    }

    public void setImageLink(String imageLink) {
        this.imageLink = imageLink;
    }
}
